### ISSUE TITLE
Subtitles are NOT adhering to the ISO rules

### ISSUE THREAD

**Issue by Trapizomba** [SickRage/SickRage#2241]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: ae848a6c2915815167b69d9662a785cb9f889781
OS: FreeBSD-10.3-RELEASE-amd64-64bit-ELF
What you did: Searched for subtitles after setting the subtitle plugin.
What happened: Subtitle downloaded
What you expected: Subtitle files renamed with ISO 639 codes.

https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

Logs:
```
NONE
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by gaetan-petit** [comment]
@Trapizomba did you check this option in the subtitles settings ?
<img width="668" alt="capture d ecran 2016-08-22 a 09 56 17" src="https://cloud.githubusercontent.com/assets/851486/17847541/c6510544-684e-11e6-85ad-90203967f5cc.png">

**Comment by Trapizomba** [comment]
@gaetan-petit
Yes, but for Brazilian Portuguese, SickRage rename the files with pt-BR wich isn't included in ISO codes...

**Comment by duramato** [comment]
There isn't any Portuguese brazilian 2 or 3 iso letter code. What do you expect..

**Comment by Trapizomba** [comment]
@duramato
I expect the respect of the ISO... But looks like it's too hard to SickRage devs accomplish this... Thank you for nothing...

**Comment by gaetan-petit** [comment]
hey @duramato maybe @Trapizomba is right, if Sickrage appends pt-BR to the filename. This is probably not attended.

**Comment by duramato** [comment]
It add pt-BR as a way to distinguish between all of the PT dialects. Otherwise it would show .pt for pt-PT and pt-BR which while true,  it isn't something that wanted becouse people use 2 at the same time.

**Comment by Trapizomba** [comment]
@duramato
People using .pt-PT will never use .pt-BR and people using .pt-BR will never use .pt-PT... Just because these langs are too different!

Another solution may be let people set the desired suffix but I know this will be a huge change... 

Anyway... The ISO definition is an official definition, so I really think SickRage should use it.

**Comment by duramato** [comment]
`People using .pt-PT will never use .pt-BR and people using .pt-BR will never use .pt-PT... Just because these langs are too different!` Wrong, i know tons of people that do it. So if they don't get a sub in one of the dialects they will from the other. As you can still understand what its written.
Also if it where to use ISO you wouldn't get subs downloaded as searching for the ISO os Portuguese would only return pt-PT.

**Comment by Trapizomba** [comment]
Sorry... But... I don't see any relation between searches and downloads... Neither downloads and suffixes...

Why not let the users set the disered suffix?

**Comment by duramato** [comment]
It is all set in a way to break anything, and for dialects to live along side.
That doesn't seem very ideal, if people set for example ISO 639 with 2 letters it would enter an infinit loop when there are 2 diferent dialects wanted in subs becouse for example it would download pt-PT , save as .pt , then pt-BR would get downloaded as .pt too overwritting the previous one and would keep searching becouse there would only be a .pt ,which is pt_PT.
Bottom line it is like this for a reason.

**Comment by Trapizomba** [comment]
Right... But why SickRage can be configured to rename episodes and can not be configured to rename subtitles? You already have the code/engine for episodes...

**Comment by duramato** [comment]
Im not following.
Renaming file name is one thing, but this isn't quite part of the file name, but something which ids the language, like extensions id the kind of file type (sort of). Subtitles need to have the same name as the video file outherwise it won't get recognized as a subtitle for that video, and im not only talking about SR.
